plan/aggregation_push_down.go

```diff
@@ -54,24 +76,14 @@ func (a *aggPushDownSolver) collectAggFuncs(agg *Aggregation, join *Join) (valid
 	leftChild := join.GetChildByIndex(0)
 	for _, aggFunc := range agg.AggFuncs {
 		if a.isDecomposable(aggFunc) {
```

Comment:
If `a.isDecomposable(aggFunc)` is false, you can return `false, nil, nil`. It will reduce nesting, and it will be clearer.